.github/workflows/check.yml

```
@@ -116,13 +116,13 @@ jobs:
     - run: pip install -r requirements-dev.txt
     - run: python3 -m pyflakes $(git ls-files | grep -E '\.(py|pyw)$')
 
-  bindcheck:
+  flake8-tkinter:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
     - uses: actions/setup-python@v4
       with:
         python-version: "3.10"
-    - run: python3 scripts/bindcheck.py porcupine
-    - run: python3 scripts/bindcheck.py more_plugins
-    - run: python3 scripts/bindcheck.py tests
+        cache: pip
+    - run: pip install flake8-tkinter==0.5.0
```

Let's also pin the flake8 version so that it can't break unexpectedly

```suggestion
    - run: pip install flake8==5.0.4 flake8-tkinter==0.5.0
```